core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -6446,3 +6446,8 @@ groups:
     #name: volume5
     - pull-ci-openshift-priv-library-go-master-verify-deps-build01
     - pull-ci-openshift-priv-library-go-master-verify-build01
+  "app.ci":
```

Comment:
Hmm ... our check did not find this out.

https://github.com/openshift/release/blob/98420605a77bee3b316866b3cc2c185a9e23b3bd/core-services/sanitize-prow-jobs/_config.yaml#L4-L6

Reply:
Yeah this kind of error is unfortunate, golang can not find it out, it will just ignore the first occurence of the string key. We need something like `yamllint` for this: https://github.com/adrienverge/yamllint

Reply:
> it will just ignore the first occurence of the string key

Does not feel right. `sanitize-job` should fail if it job's cluster stanza is changed (specially this is app.ci and the defailt one is api.ci).